content/SCALE/SCALETutorials/Shares/SMB/SMBMultichannel.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Setting Up SMB Multichannel"
+description: "This article provides information on setting up SMB Multichannel."
+weight: 60
+aliases:
+tags:
+ - scalesmb
+---
+
+{{< toc >}}
+
+SMB Multichannel allows servers to use multiple network connections simultaneously by combining the bandwidth of several Network Interface Cards (NICs) for better performance.
+
+{{< hint warning >}}
+SMB Multichannel does not function if you combine NICs into a LAGG. 
+{{< /hint >}}
+
+## Activate Multichannel in TrueNAS Scale
```

Comment:
It is not incorrect to use Activate, but it reads more like a procedure step. Using gerunds makes it more like a description of the content in the section so I suggest using the ing form. If you want the reader to execute in a particular order, then I suggest using an introduction heading with the procedure overview and list the list the steps in sequence and you can do internal links to each section.
```suggestion
## Activating Multichannel in TrueNAS Scale
```